crates/bevy_reflect/bevy_reflect_derive/src/utility.rs

```diff
@@ -0,0 +1,23 @@
+//! General-purpose utility functions for internal usage within this crate.
+
+use bevy_macro_utils::BevyManifest;
+use proc_macro2::{Ident, Span};
+use syn::Path;
+
+/// Returns the correct path for `bevy_reflect`.
+pub(crate) fn get_bevy_reflect_path() -> Path {
+    BevyManifest::just_get_path("bevy_reflect")
+}
```

Comment:
@alice-i-cecile How is this?

> Also, not sure if `just_get_path` is the best name or not, but I couldn't think of anything else haha

Reply:
IMO just remove this function, and call the wrapped method directly in the code. I don't think the extra layer adds anything here: it's only used once.